ui/src/components/MainContainer.js

```diff
@@ -213,6 +214,12 @@ const MainContainer = () => {
           permissions={['admin', 'stewardship']}
           component={AdminContainer}
         />
+        <PrivateRoute
```

Comment:
mayb remove this first. Kris is still looking at how to handle password protected pages